src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -510,6 +510,46 @@ def get_image_config(args):
     return image
 
 
+def prep_rhel6(args):
+    """
+    Extra treatment for rhel/centos6.
+
+    1) Create openssl_el6.conf in $HOME/.config and set OPENSSL_CONF to it.
+    2) Set the ssh extra args to ansible_args for the ansible execution
+       as well as to the environment variable TEST_EXTRA_SSH_ARGS for qemu.
+    """
+    if args.image_name:
+        image_name = args.image_name
+    else:
+        image_name = os.path.splitext(os.path.basename(args.image_file))[0]
+    if not image_name.endswith("-6"):
+        return
+    # If it does not exist, create openssl_el6.conf in ~/.config.
+    openssl_el6_conf = os.path.join(
+        os.environ["HOME"], ".config", "openssl_el6.conf"
+    )
+    os.environ["OPENSSL_CONF"] = openssl_el6_conf
+    if not os.path.exists(openssl_el6_conf):
+        s = """\
+            .include /etc/ssl/openssl.cnf
```

Comment:
Does this file exist on rhel7?

Reply:
looks like this file is provided by ca-certificates - on rhel7 it is `/etc/pki/tls/openssl.cnf`

Reply:
> looks like this file is provided by ca-certificates - on rhel7 it is `/etc/pki/tls/openssl.cnf`

Thank you, @richm! On rhel-8 and rhel-9, it is `/etc/pki/tls/openssl.cnf`, too. So, I should check `/etc/pki/tls/openssl.cnf` first and if it exists, we'll use it. Actually, it is `/etc/pki/tls/openssl.cnf` on fedora, too. `/etc/ssl/openssl.cnf` is a symlink. :p